dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -30,6 +30,7 @@ FROM {{ .from }}
 # Contains the elastic agent image variant, an empty string for the standard variant
 # or "complete" for the bigger one.
 ENV ELASTIC_AGENT_IMAGE_VARIANT={{.Variant}}
+ENV BEAT_LOCAL_USER={ .user }
```

Comment:
Could we call this just `LOCAL_USER`? I assume it could apply to any subprocess, it does not have to be a Beat? Or `LOCAL_USER_SUBPROCESS`?

Reply:
I've renamed it to `BEAT_SETUID_AS` I think it should be obscure and not something with any risk of clashing.